modules/reactor/doc/content/source/meshgenerators/PatternedHexPeripheralModifier.md

```diff
@@ -0,0 +1,88 @@
+# PatternedHexPeripheralModifier
+
+!syntax description /Mesh/PatternedHexPeripheralModifier
+
+## Overview
+
+This `PatternedHexPeripheralModifier` class utilizes [`TransitionLayerTools`](/TransitionLayerTools.md) to replace the outmost layer of the quad elements of the hexagonal assembly mesh generated by [`PatternedHexMeshGenerator`](PatternedHexMeshGenerator.md) (or its derived class [`HexIDPatternedMeshGenerator`](HexIDPatternedMeshGenerator.md)) with a transition layer consisting of triangular elements so that the assembly mesh can have nodes on designated positions on the external boundary. This boundary modification facilitates the stitching of hexagonal assemblies which have different node numbers on their outer periphery due to differing numbers of interior pins and/or different azimuthal discretization.
```

Comment:
this must mention this is in 2D

Reply:
`PatternedHexMeshGenerator` itself is 2D. Will add mentioning of 2D anyway.